src/store/session/index.ts

```diff
@@ -12,6 +12,7 @@ export const state: SessionState = {
   rLoginInstance: undefined,
   txType: undefined,
   balance: new WeiBig('0', 'wei'),
+  btcAccount: '',
```

Comment:
Sorry for being nitpicky but we should refer to `btcAddress` rather than account